### DISCUSSION TITLE
Ability to force mount actions

### DISCUSSION THREAD

**Discussion by IllesAprod** [filamentphp/filament#5381] [Ideas]
Hi!

When creating a custom filament page, it would be handy to add actions that can be mounted by custom buttons and don't appear in the top right corner.

Right now, if I set the action to disabled, it can't be used with $this->mountAction()

Can we add a second parameter to the mountAction() function which indicates if the action should be force mounted?

```
mountAction($name, $force = false)
```

I think it wont be a breaking change.

Thanks!

**Comment by danharrin** [comment]
This is possible in Filament v3 - https://github.com/filamentphp/filament/blob/3.x/packages/actions/docs/05-adding-an-action-to-a-livewire-component.md
